commands/command.go

```diff
@@ -32,28 +33,11 @@ type Marshaler func(Response) (io.Reader, error)
 // (or an error on failure)
 type MarshalerMap map[EncodingType]Marshaler
 
-// HelpText is a set of strings used to generate command help text. The help
-// text follows formats similar to man pages, but not exactly the same.
-type HelpText struct {
-	// required
-	Tagline               string            // used in <cmd usage>
-	ShortDescription      string            // used in DESCRIPTION
-	SynopsisOptionsValues map[string]string // mappings for synopsis generator
-
-	// optional - whole section overrides
-	Usage           string // overrides USAGE section
-	LongDescription string // overrides DESCRIPTION section
-	Options         string // overrides OPTIONS section
-	Arguments       string // overrides ARGUMENTS section
-	Subcommands     string // overrides SUBCOMMANDS section
-	Synopsis        string // overrides SYNOPSIS field
-}
-
 // Command is a runnable command, with input arguments and options (flags).
 // It can also have Subcommands, to group units of work into sets.
 type Command struct {
```

Comment:
why didnt command get extracted?

Reply:
because we still have all these legacy commands. They still import the old lib, but can be used as subcommands in the new lib.